packages/contractkit/src/utils/send-tx.ts

```diff
@@ -0,0 +1,56 @@
+import debugFactory from 'debug'
+import { Address } from 'src/base'
+import { toTxResult, TransactionResult } from 'src/utils/tx-result'
+import { TransactionObject } from 'web3/eth/types'
+
+const debug = debugFactory('contractkit:sendtx')
+
+export interface TxOptions {
+  gasInflationFactor?: number
+  gasBeneficiary?: Address
+  gasCurrency?: Address
+  from?: Address
+  estimatedGas?: number | undefined
+}
+
+/**
+ * sendTransactionAsync mainly abstracts the sending of a transaction in a promise like
```

Comment:
can call this `sendTransaction` now. Can also remove the TODO below and reference to sendTransactionFactory probably

Reply:
sure, forgot to do that when moving it